modules/ROOT/pages/advanced/creating-an-icon-pack.adoc

```diff
@@ -127,11 +130,13 @@ An icon pack can be served either:
 * <<deploytheiconpackwithtinymce,With {productname}>>
 * <<deploytheiconpackandtinymceseparately,Separate from {productname}>>
```

Comment:
These legacy xrefs need converting to the more modern `xref:` syntax.

Reply:
converted I'll convert all old anchor syntax in the context of this PR

Reply:
Ok, I should have added it to all anchors